features/base/exec.post

```diff
@@ -30,3 +30,5 @@ rm "$rootfs/etc/.resolv.conf.systemd-resolved.bak"
 fi
 
 find "$rootfs/var/log/" "$rootfs/var/cache/" -type f -delete
+
+echo > "$rootfs/etc/hostname"
```

Comment:
I think you'll need to do it the same as in the `server/exec.post` used to do it via

```
rm "$rootfs/etc/hostname"
echo -n > "$rootfs/etc/hostname"
```

Reason: `/etc/hostname` *may* be a symlink, in that case you'll write to the symlink destination, not replace the file with an empty one.